docs/mecsimcalc-library.md

```diff
@@ -653,3 +653,112 @@ Displaying Image
+| **`spreadsheet_id`**       | **str**  | The unique identifier of the target Google Spreadsheet.                     |
+| **`values`**               | **list of lists** | The data to append. Each list element represents a row of data.                                                                              |
+| **`range_name`**           | **str** (optional)  | The A1 notation of the range to start appending data (Defaults to 'Sheet1!A1').                                                              |
+| **`include_timestamp`**    | **bool** (optional) | If True, appends the current timestamp to each row of data (Defaults to True).                                                               |
+
+#### Returns:
+
+| Return Type | Description                              |
+|-------------|------------------------------------------|
+| **`dict`**  | The response from the Google Sheets API, containing details of the append operation. |
+
+#### Example:
+
+#### Code step:
+
+```python
+from datetime import datetime
+import mecsimcalc as msc
+
+def main(inputs):
+    service_account_info = {
+        # Your service account info here
+    }
+    spreadsheet_id = 'your_spreadsheet_id_here'
+    values = [
+    [ inputs['input_1'], inputs['input_2'], inputs['input_3'] ],
+    ]    
+    result = msc.append_to_google_sheet(service_account_info, spreadsheet_id, values, include_timestamp)
+
+```
+
+
+### send_gmail
+
+[**[Source]**](https://github.com/MecSimCalc/MecSimCalc-utils/blob/v0.1.6/mecsimcalc/quiz_utils.py#L124)
+
+```python
+send_gmail(
+	sender_email="123@gmail.com", 
+	receiver_email="456@xxx.ca", 
+	subject="Quiz", 
+	app_password = "xxxx xxxx xxxx xxxx", 
+	values = [("name", "grade")]
+)
+```
+#### Description:
+
+This function sends an email with specified values formatted in the message body, utilizing a service account for authentication.
+
+#### Arguments:
+
+| Argument            | Type     | Description                                                                                                          |
+|---------------------|----------|----------------------------------------------------------------------------------------------------------------------|
+| **`sender_email`**      | **str**  | The email address of the sender.                                                                                     |
+| **`receiver_email`**    | **str**  | The email address of the receiver.                                                                                   |
+| **`subject`**           | **str**  | The subject line of the email.                                                                                       |
+| **`app_password`**      | **str**  | The app-specific password for the sender's email account.  |
+| **`values`**            | **list** | A list of tuples. Each tuple contains data to be included in the email body.                                         |
```

Comment:
Maybe include more info here. Seems a bit confusing. Like are tuples in the same paragraph?